extension/src/paymentHandler/payment-utils.js

```diff
@@ -59,6 +59,19 @@ function createChangeTransactionStateAction(transactionId, transactionState) {
   }
 }
 
+function createChangeTransactionTimestampAction(
+  transactionId,
+  paymentObjTimestamp
+) {
+  let currentTimestamp = new Date().toISOString()
+  if (paymentObjTimestamp !== undefined) currentTimestamp = paymentObjTimestamp
+  return {
+    action: 'changeTransactionTimestamp',
+    transactionId,
+    timestamp: currentTimestamp,
+  }
```

Comment:
A bit shorter and maybe a bit easier to read:

```
  return {
    action: 'changeTransactionTimestamp',
    transactionId,
    timestamp: (paymentObjTimestamp == undefined) ? new Date().toISOString() : paymentObjTimestamp,
  }
```

Reply:
Undo the timestamp assignment in extension module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2